compiler/optimizer/abstractinterpreter/IDT.cpp

```diff
@@ -39,6 +39,7 @@ void TR::IDT::print()
 
    if (!verboseInlining && !traceBIIDTGen)
       return;
+   TR_VerboseLog::vlogAcquire();
```

Comment:
The vlog lock only needs to be acquired and released when `verboseInlining` is true

Reply:
The acquire and release for the lock has been added when verboseInlining is true